Formula/kt-connect.rb

```diff
@@ -0,0 +1,32 @@
+class KtConnect < Formula
+  desc "Toolkit for integrating with kubernetes dev environment more efficiently"
+  homepage "https://alibaba.github.io/kt-connect"
+  url "https://github.com/alibaba/kt-connect/archive/refs/tags/v0.3.5.tar.gz"
+  sha256 "b1a2f0f71feb7d8c4418f73048c24687d513dc1cabb1f68ffac3be5baa0e3f5a"
+  license "GPL-3.0-or-later"
```

Comment:
let's also add head build support
```suggestion
  license "GPL-3.0-or-later"
  head "https://github.com/alibaba/kt-connect.git", branch: "master"
```